cognite/client/data_classes/data_modeling/views.py

```diff
@@ -446,26 +479,117 @@ def as_apply(self) -> SingleHopConnectionDefinitionApply:
             description=self.description,
             edge_source=self.edge_source,
             direction=self.direction,
+            connection_type=self.connection_type,
         )
 
 
 @dataclass
-class ConnectionDefinitionApply(ViewPropertyApply):
-    ...
+class ReverseSingleHopConnection(ConnectionDefinition):
+    """Describes the direct relation(s) pointing to instances read through this view. This connection type is used to
+    aid in discovery and documentation of the view
+
+    It is called 'ReverseDirectRelationConnection' in the API spec.
+
+    Args:
+        source (ViewId): The node(s) containing the direct relation property can be read through
+            the view specified in 'source'.
+        through (PropertyId): The view or container of the node containing the direct relation property.
```

Comment:
`NodeResultExpression.through` is a `list[str] | tuple[str, str, str]`. Let's just do that here aswell to stay consistent.

Reply:
Do you mean we'd use the array length to reason if we're dealing with a view or container reference, or reusing the logic in DMS to reference a view as [space, externalId/version, property]? (compared with [space, externalId, version, property]? Personally I never liked that check if id[1] contains '/'.

Reply:
View and container references have the same length - views are (space, extid/version, prop) while for containers it's (space, extid, prop). 

But it's to be consistent within the SDK, and property references are lists elsewhere. Accepting both forms is fine with me though.